src/mui/detail/RecordForm.js

```diff
@@ -0,0 +1,80 @@
+import React, { PropTypes } from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { Toolbar, ToolbarGroup } from 'material-ui/Toolbar';
+import { SaveButton } from '../button';
+import { Labeled } from '../input';
+
+let validators = {};
+
+const validate = (values) => {
```

Comment:
This is an impure function. Can't currently see how to purify this one. Working on it. Any idea @fzaninotto or @djhi?